### PR TITLE
Add option for HD cover

### PR DESCRIPTION
**Pull request by pfei-sa**
Change cover_url if user wants a HD cover

### CODE REVIEW

**Comment by bebound** [comment]
Thanks for the commit, but I can't merge it, there are two minor problems in this commit:
1 epub.py, line113, wrong indent
2 novel.py line233, needless debug info.

Also, what's the purpose of the _HEADERS_BACKUP?

I think hd cover is a good idea, it could be a default setting. Thanks.

**Comment by pfei-sa** [comment]
I will double check it ASAP

On my computer, sometimes it will time out when downloading the
last(HD-cover)
If it time out, switch to another header and download again fixed the
problem.
BTW, what's your python and other library's versions? It's always good to
know that.

File: epub.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 _download_queue = queue.Queue()
 _PROGRESS_LOCK = threading.Lock()
 _HEADERS = {'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; WOW64; rv:19.0) Gecko/20100101 Firefox/19.0'}
-
+_HEADERS_BACKUP = {'User-Agent': 'Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36'}
```
Comment (pfei-sa) [inline epub.py]: If timeout, change UA

**Comment by pfei-sa** [comment]
Fixed problem. If you want to make it default, just do so. Some time when I try to generate book with HD cover it stuck. By changing UA my problem was solved. BTW I am not in China so maybe _HEADERS_BACKUP is not necessary.

**Comment by bebound** [comment]
Thank you, I'll check if the backup headers is necessary. I'm also not in China now.

**Comment by pfei-sa** [comment]
Also I didn't make it default. If you want it to be default, please do so. This is my first time working with an open source project on Github, thank you for pointing my error out.

**Comment by bebound** [comment]
No worries, I'm also a noob.

**Pull request merged**